File: GarageWizardClient/src/app/models/reservation.ts
export class Reservation
{
    spotID!: string;
    vehicleID!: string;
    service: number = 0;

    constructor(sID: string, vID: string, service: number){
        this.spotID = sID;
        this.vehicleID = vID;
        this.service = service;
    }
}

File: GarageWizardAPI/GarageCoreMVC/Models/Reservation.cs
using GarageCoreMVC.Common.Enums;

namespace GarageCoreMVC.Models
{
    public class Reservation
    {
        public string? spotID {  get; set; }
        public string? vehicleID { get; set; }
        public int service {  get; set; }
    }
}
